vertx-web/src/main/java/io/vertx/ext/web/sstore/impl/SessionImpl.java

```diff
@@ -53,17 +50,25 @@
   private static final byte TYPE_SERIALIZABLE = 12;
   private static final byte TYPE_CLUSTER_SERIALIZABLE = 13;
 
+  private final PRNG prng;
+
   private String id;
   private long timeout;
   private Map<String, Object> data;
```

Comment:
The field is final see: http://stackoverflow.com/questions/14790478/final-vs-volatile-guaranntee-w-rt-to-safe-publication-of-objects